src/Rings/binomial_ideals.jl

```diff
@@ -672,7 +672,7 @@ Given a cellular binomial ideal `I`, return the associated primes of `I`.
 The result is defined over the abelian closure of $\mathbb Q$. In the output, if needed, the generator for roots of unities is denoted by `zeta`. So `zeta(3)`, for example, stands for a primitive third root of unity.
 
 # Examples
-```jldoctest
+```jldoctest; filter = r"(coeff \d+ already initialized\n|)"
```

Comment:
If there is non-deterministic verbose Singular.jl output, shouldn't this be an issue there to get fixed?